internals/cli/cmd_enter_test.go

```diff
@@ -231,3 +232,40 @@ func (s *PebbleSuite) TestEnterHelpCommandHelpArg(c *C) {
 	c.Check(stdout, Matches, "(?s).*\\bThe help command displays information about commands\\..*")
 	c.Check(exitCode, Equals, 0)
 }
+
+// TestEnterSubCommandWaits checks that the subcommand in enter
+// starts **after** the default services have started. It utilizes
+// the modification time of /proc/[PID] directory for checking.
+func (s *PebbleSuite) TestEnterSubCommandWaits(c *C) {
+	layerTemplate := dumbDedent(`
+		services:
+		  stat:
+		    override: replace
+		    command: /bin/sh -c 'stat -c "%y" /proc/$$ > $PEBBLE/stat; sleep 1;'
```

Comment:
Just curious; why the `sleep 1`?

Reply:
To avoid getting the new error report this PR introduced:
```
cannot run pebble: error starting default services: cannot perform the following tasks:
- Start service "stat" (cannot start service: exited quickly with code 0)
```
This reminds me, I should add a test for that as well. Thank you.

Reply:
Added in https://github.com/canonical/pebble/pull/257/commits/1281c967da6093bb3e2e6567a792f83dd860a079. :v: